Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,205 @@
 
+var alphabet = [String]()
+alphabet = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"]
+
+var isPlaying = true
+
+repeat {
+    
+    
+    let randomWord = allTheWords.randomElement()
+    let word = randomWord ?? ""
+
+    //var charNum = word.count
+    var letterGuessed = Set<String>()
+    var letterArray = [String]()
+
+    print()
+
+    for _ in word {
+//        print(char, terminator: " ")
+        letterArray.append("_")
+    }
+
+    print()
+
+    print("""
+LET'S PLAY HANGMAN, 🤩🤩🤩!!!
+
+    +____________
+    |          |
+    |
+    |
+    |
+    |
+____|_______________________________________
+
+""")
+    print()
+    print("Guess a letter!!!")
+    print()
+    var attemptsCount = 0
+//    var correctGuessCount = 0
+    var correctGuessCount = 0
+
+    while attemptsCount < 6 {
+        
+        for char in letterArray {
+            print(char, terminator: " ")
+        }
+        print()
+        let guessLetter = readLine() ?? ""
+        let guess = guessLetter.lowercased()
+        
+        if guess.count > 1 || !alphabet.contains(guess){
```

Comment:
This was a brilliant way to account for the user entering more than one character or anything outside of the alphabet.